docs/user-guide.md

```diff
@@ -508,6 +508,12 @@ performed.
     to a string formed by concatenating the version number, build type and build
     string.
 
+*   `WARMBOOT_ENABLE_DCACHE_EARLY` : Boolean option to enable D-cache early on
+    the CPU after warm boot. This is applicable for platforms which do not
+    require interconnect programming to enable cache coherency (eg: single
+    cluster platforms). If this option is enabled, then warm boot path
+    enables D-caches immediately after enabling MMU. This option defaults to 0.
+
```

Comment:
minor: I would prefer if the description for HW_ASSISTED_COHERENCY were updated to say if HW_ASSISTED_COHERENCY is enabled then this implies WARMBOOT_ENABLE_DCACHE_EARLY is also enabled.